client/scss/components/body.scss

```diff
@@ -0,0 +1,4 @@
+.body {
```

Comment:
This should be inside `base/base.scss` and the class name should be more relevant (e.g. `.content-column`).